requirements.txt

```diff
@@ -1,3 +1,4 @@
 ansible==2.5.8
 awscli==1.16.10
 boto3==1.8.5
+botocore>=1.11.5,<1.12.0
```

Comment:
I believe botocore doesn't need to be declared here since it's a sub-dependency.